docs/permission/plugin-authors/02-adding-a-basic-permission-check.md

```diff
@@ -117,24 +115,28 @@ That's it! Now your plugin is fully configured. Let's try to test the logic by d
 
 ## Test the authorized create endpoint
 
+Before running this step, please make sure you followed the steps described in [Getting started](../getting-started) section.
+
 In order to test the logic above, the integrators of your backstage instance need to change their permission policy to return `DENY` for our newly-created permission:
 
 ```diff
 // packages/backend/src/plugins/permission.ts
 
 - import { IdentityClient } from '@backstage/plugin-auth-node';
-+ import { BackstageIdentityResponse, IdentityClient } from '@backstage/plugin-auth-node';
++ import {
++   BackstageIdentityResponse,
++   IdentityClient
++ } from '@backstage/plugin-auth-node';
   import {
     PermissionPolicy,
-+   PolicyAuthorizeQuery,
-    PolicyDecision,
++   PolicyQuery,
   } from '@backstage/plugin-permission-node';
 
-- class AllowAllPermissionPolicy implements PermissionPolicy
-+ class MyPermissionPolicy implements PermissionPolicy {
+- class TestPermissionPolicy implements PermissionPolicy
++ class TestPermissionPolicy implements PermissionPolicy {
```

Comment:
Since the class name doesn't change here we can remove the `-` and `+` from these lines